### PR TITLE
Draft proposal - yarn install action

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request adds temporary note for local contributors calling for them to run yarn install.

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
-## Purpose of this pull request
+## PLEASE READ
+
+Following a recent back-end update, all contributors with a _local clone of our repository_ are required to run `yarn install`. This does not apply to forked pull requests.
```
Comment (jpipkin1) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: @kimsauce, it says, "This does not apply to forked pull requests." But we've had problems with forked repos too. Are you sure they don't need to run `yarn install` in forked repos?

Comment (kimsauce) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).
```

Comment (kimsauce) [reply .github/PULL_REQUEST_TEMPLATE.md]: You're correct! It does apply to those with physical, local forks of our repo, but doesn't apply to those who follow the ["Edit this page" workflow](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits), as that doesn't involve cloning/forking locally. Can you take a look at this revised wording when you can?

**Review by jpipkin1 — APPROVED** [review]
Looks good, @kimsauce.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jpipkin1) [reply .github/PULL_REQUEST_TEMPLATE.md]: Looks good.

**Pull request merged**